Tasks/DotNetCoreCLIV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -103,7 +103,7 @@
   "loc.messages.dotnetCommandFailed": "Dotnet command failed with non-zero exit code on the following projects : %s",
   "loc.messages.noProjectFilesFound": "Project file(s) matching the specified pattern were not found.",
   "loc.messages.noPublishFolderFoundToZip": "A publish folder could not be found to zip for project file: %s.",
-  "loc.messages.noWebProjectFound": "No web project was found in the repository. Web projects are identified by presence of either a web.config file or wwwroot folder in the directory. You can set Publish Web Projects property to false (publishWebProjects: false in yml) if your project doesn't follow this convention or if you want to publish projects other than web projects.",
+  "loc.messages.noWebProjectFound": "No web project was found in the repository. Web projects are identified by presence of either a web.config file, wwwroot folder in the directory, or by the usage of Microsoft.Net.Web.Sdk in your csproj file. You can set Publish Web Projects property to false (publishWebProjects: false in yml) if your project doesn't follow this convention or if you want to publish projects other than web projects.",
```

Comment:
Should you just say 'project file' as it can be fsproj etc also

Reply:
Done